backend/src/main/java/com/trackpack/app/model/joining/UserParcel.java

```diff
@@ -0,0 +1,36 @@
+package com.trackpack.app.model.joining;
+
+import com.trackpack.app.keys.UserParcelKey;
+import com.trackpack.app.model.tracking.ShipmentTracking;
+import com.trackpack.app.model.user.User;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.time.OffsetDateTime;
+
+@Entity
+@Table(name = "UserParcel")
+@Data
+@NoArgsConstructor
+public class UserParcel {
+
+    @EmbeddedId
+    UserParcelKey id;
+
+    @ManyToOne
+    @MapsId("userId")
+    @JoinColumn(name = "userId")
+    User user;
```

Comment:
Is there any cases we need user object on UserParcel? Maybe it's better to have just ID reference to User and we'll fetch him separately when it's needed?